### ISSUE TITLE
Can't start via systemd

### ISSUE THREAD

**Issue by levinus** [haiwen/seahub#1656]
Hi,
as I wrote [here](https://forum.seafile.com/t/cant-start-via-systemd) in the forums I can't start seafile via systemd or similar services. I create this issue in seahub because I think the problem is with seahub.
When I start seafile & seahub via systemd (using the `.service` files provided from the manual) I don't get any error messages, everything seems fine. I am able to use SeafDAV but when I try to use seahub I only get a 'Page unavailable' message. Seahub runs fine if I log into the seafile user manually with `su seafile` and then execute the start commands.
When I look into the logs [this error message](https://hastebin.com/uwoyigimut.sql) is printed in the seahub django log. Also other tables than `django_session` appear in the log with similar error messages.
The error message is strange because the django backend for SQLite appears in the stacktrace but I do not use SQLite and specified a MySQL/MariaDB backend in my `seahub_settings.py` (which is used when starting manually). In my mysql DB the tables exist but clearly not in a non-existent SQLite DB.
Could it be that seahub ignores my `seahub_settings.py` file if started via systemd?
I tried to delete the `seahub_settings.pyc` file because in my understanding the file is newly generated by seahub when its not there anymore. It wasn't generated new when started via systemd.

I'm running a Ubuntu 16.04 Server with seafile 6.1.1

Greetings,
Levin

**Comment by shoeper** [comment]
This is very unlikely a Seafile and very likely a usage problem of systemd.

First of all I'd recommend removing the /bin/bash.

And then I'd close this issue as the forum is the correct place to discuss it.

**Comment by levinus** [comment]
As I already said (wrote in the forum and in the summary in the issue description) I now use exactly the template from the seafile manual (with customized paths and fastcgi for seahub, I'm runnning seahub behind a apache reverse proxy). Without /bin/bash.
If this is a configuration issue the issue is also in the seafile manual.
btw: I tried this some time ago with a script in /etc/init.d that uses sudo to run seafile/seahub as user seafile and the same error message came up.
It seemed like no one in the forum could help me so I posted here.
